wee_slack.py

```diff
@@ -2248,6 +2255,11 @@ def process_presence_change(message_json, eventrouter, **kwargs):
         user = kwargs["user"]
         team = kwargs["team"]
         team.update_member_presence(user, message_json["presence"])
+    if "users" in message_json:
+        team = kwargs["team"]
+        for user_id in message_json["users"]:
+            user = self.teams[team].users[user_id]
```

Comment:
`self` isn't defined here. You should just use `team`.

Reply:
✓